### PR TITLE
#122 standalone cluster manager

### PR DESCRIPTION
**Pull request by gyustorm**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

File: sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java [hunk 1]
```diff
@@ -199,6 +199,9 @@ protected void initialize(String path, List<String> columns) throws IOException
     config.set("spark.sql.parquet.int96AsTimestamp", "false");
 
     this.file = new Path(path);
+    System.out.println("READ_RETRY before getFileStatus " + path);
```
Comment (zheniantoushipashi) [inline sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java:202]: 换成 logInfo 要么去掉

File: sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala [hunk 2]
```diff
@@ -239,7 +239,13 @@ class SQLAppStatusListener(
         return
       }
 
-      val updates = accumUpdates
+      val updates = if (skewDetectEnabled) {
+        import InternalAccumulator._
+        accumUpdates.filter { acc =>
+          acc.update.isDefined && (metrics.accumulatorIds.contains(acc.id) ||
```
Comment (zheniantoushipashi) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala:245]: 之前改动的一致性 和 数据倾斜的怎么都混到standalone 里面来了

Comment (gyustorm) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala:245]: 在给appzen的hotfix分支，standalone 发布的话需要cherry-pick过来，不然会有问题

Comment (gyustorm) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala]: 已拆开提交

File: sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java (hunk 1 above)
Comment (gyustorm) [reply sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java]: 已移除

**Pull request merged**